kie-api/src/main/java/org/kie/api/event/process/ProcessEventListener.java

```diff
@@ -92,4 +92,12 @@
      */
     void afterVariableChanged(ProcessVariableChangedEvent event);
 
+    /**
+	 * This listener method is invoked right before a node in a process instance is being removed from the process instance
+     * @param event
+     */
+    default void beforeNodeRemoved(ProcessNodeTriggeredEvent event) {
+        // empty for backwards compatibility
+    }
```

Comment:
shouldn't we have afterNodeRemoved as well to be consistent that each event has both before and after methods?